server/initializers/forms.js

```diff
@@ -387,7 +392,8 @@ function formOptions (form) {
       apiData: generateApiData(form.fields),
       apiUpdate: generateApiUpdate(form.fields),
       exportData: generateExportData(form)
-    }
+    },
+    hooks: form.hooks
```

Comment:
I know this is used in this PR, but it is better as a separate PR and could be merged in master sooner